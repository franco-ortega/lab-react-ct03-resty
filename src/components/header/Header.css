.Header {
    background-color: black;
    color: white;
    padding: 20px;
    width: 100vw;
}
