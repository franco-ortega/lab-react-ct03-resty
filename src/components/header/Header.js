import React from 'react';
import styles from './Header.css';

export default function Header() {
  return (
    <h1 className={styles.Header}>
        RESTy
    </h1>
  );
}
