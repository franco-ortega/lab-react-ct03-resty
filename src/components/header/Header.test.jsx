import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders Header presenational component', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment).toMatchSnapshot();
  });
});
