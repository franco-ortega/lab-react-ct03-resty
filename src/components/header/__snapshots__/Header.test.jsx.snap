// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header presenational component 1`] = `[Function]`;
